testserver/binaries.go

```diff
@@ -120,13 +120,13 @@ func downloadLatestBinary() (string, error) {
 		if info.Mode().Perm() == finishedFileMode {
 			return localFile, nil
 		}
-		log.Printf("waiting for download of %s", localFile)
+		log.Printf("testserver: waiting for download of %s", localFile)
 		time.Sleep(time.Millisecond * 10)
 	}
 
 	if err := downloadFile(response, localFile); err != nil {
 		if err := os.Remove(localFile); err != nil {
-			log.Printf("failed to remove %s: %s", localFile, err)
+			log.Printf("testserver: failed to remove %s: %s", localFile, err)
```

Comment:
we should wrap the error here rather than use fmt.Errorf

see https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/1676640931/Error+handling+basics
(although for this library we cannot use cockroachdb/errors, so we can use the golang errors package